dxm/src/main/java/io/pcp/parfait/dxm/MmvWritable.java

```diff
@@ -2,8 +2,9 @@
 
 import java.nio.ByteBuffer;
 
-public interface MmvWritable {
+interface MmvWritable {
 
+    int getOffset();
```

Comment:
It ended up there as it was also used as part of the MMV writing (setting up the TOC offsets). It turns out we don't have to have this on the interface or really the `MmvWritable` interface at all. 

There might be another missing abstraction which is the sections themselves. The `PcpMmvWriter` deals with sections in a fairly procedural manner.